### DISCUSSION TITLE
STC.Miner x3

### DISCUSSION THREAD

**Discussion by sajadbaradaran** [starcoinorg/starcoin#2866] [General]
Hi, I bought Miner x3 and I want to extract STC. This is truly one of the best digital currencies. I can not track my miner. What's the problem? Miner x3 can extract stc

**Comment by jiangydev** [comment]
Miner x3 doesn't support for STC mining.

**Reply by sajadbaradaran** [reply]
Why do I extract stc currency in f2pool pool but my hash rate is zero after half an hour

**Reply by jiangydev** [reply]
Because the committed hash is invalid.
